data.sql

```diff
@@ -1,5 +1,7 @@
-/* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+		('Agumon','2020-02-03', 0, true, 10.23),
+		('Gabumon', '2018-11-15', 2, false, 8),
+		('Pikachu', '2021-01-07', 1, true, 15.04),
+		('Devimon', '2017-05-12', 5, true, 11);
+        
```

Comment:
- [ ] Great job so far 👍🏻 However, let's make sure we follow the project description 👇🏻 the second and third inserts have a wrong neutered value. Kindly fix this 💯 👍🏻 

![18](https://user-images.githubusercontent.com/79058364/198284183-842645e1-b7ee-4762-9390-6681c8d8051b.png)

- Your code should look like this: 

```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Gabumon', '2018/11/15', '2', 'true', '8.0');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Pikachu', '2021/01/07', '1', 'false', '15.04');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Devimon', '2017/05/12', '5', 'true', '11.0');
```